app/modules/entities/components/editor/lib/external_ids_url_formatters.js

```diff
@@ -1,18 +1,63 @@
+// Formatter URLs can be found on Wikidata, see https://www.wikidata.org/wiki/Property:P1630
 export const externalIdsUrlFormatters = {
+  'wdt:P213': id => `https://isni.oclc.org/xslt/DB=1.2/CMD?ACT=SRCH&IKT=8006&TRM=ISN%3A${id}`,
+  'wdt:P214': id => `https://viaf.org/viaf/${id}/`,
+  'wdt:P227': id => `https://d-nb.info/gnd/${id}`,
+  'wdt:P243': id => `https://www.worldcat.org/oclc/${id}`,
+  'wdt:P244': id => `https://id.loc.gov/authorities/${id}`,
   'wdt:P268': id => `https://catalogue.bnf.fr/ark:/12148/cb${id}`,
+  'wdt:P269': id => `https://www.idref.fr/${id}`,
+  'wdt:P349': id => `https://id.ndl.go.jp/auth/ndlna/${id}`,
+  'wdt:P496': id => `https://orcid.org/${id}`,
   'wdt:P648': id => {
     const lastLetter = id.slice(-1)[0]
     const section = openLibrarySectionByLetter[lastLetter]
     return `https://openlibrary.org/${section}/${id}`
   },
+  'wdt:P675': id => `https://books.google.com/books?id=${id}`,
+  'wdt:P906': id => `https://libris.kb.se/auth/${id}`,
+  'wdt:P950': id => `https://datos.bne.es/resource/${id}`,
+  'wdt:P1006': id => `https://data.bibliotheken.nl/doc/thes/p${id}`,
+  'wdt:P1025': id => `https://www.sudoc.fr/${id}`,
+  'wdt:P1044': id => `https://swb.bsz-bw.de/DB=2.1/PPNSET?PPN=${id}&INDEXSET=1`,
+  'wdt:P1085': id => `https://www.librarything.com/work/${id}`,
+  'wdt:P1143': id => `https://catalogo.bn.gov.ar/F/?func=direct&doc_number=${id}&local_base=GENER`,
+  'wdt:P1144': id => `https://www.loc.gov/item/${id}/`,
+  'wdt:P1182': id => `http://libris.kb.se/bib/${id}`,
+  'wdt:P1274': id => `https://www.isfdb.org/cgi-bin/title.cgi?${id}`,
+  'wdt:P1292': id => `https://d-nb.info/${id}`,
+  'wdt:P1844': id => `https://catalog.hathitrust.org/Record/${id}`,
+  'wdt:P1960': id => `https://scholar.google.com/citations?user=${id}`,
+  'wdt:P1982': id => `https://www.animenewsnetwork.com/encyclopedia/people.php?id=${id}`,
+  'wdt:P1983': id => `https://www.animenewsnetwork.com/encyclopedia/company.php?id=${id}`,
+  'wdt:P1984': id => `https://www.animenewsnetwork.com/encyclopedia/manga.php?id=${id}`,
   'wdt:P2002': id => `https://twitter.com/${id}`,
+  'wdt:P2003': id => `https://www.instagram.com/${id}/`,
   'wdt:P2013': id => `https://www.facebook.com/${id}`,
-  'wdt:P2003': id => `https://www.instagram.com/$${id}`,
   'wdt:P2397': id => `https://www.youtube.com/channel/${id}`,
+  'wdt:P2607': id => `https://bookbrainz.org/author/${id}`,
```

Comment:
`P2607` (Bookbainz Id) does not have human readable string name
![2023-12-18_10-35](https://github.com/inventaire/inventaire-client/assets/5363918/61933f88-69c5-49db-924b-124d1048816a)

Reply:
fixed by https://github.com/inventaire/inventaire-i18n/commit/1929c29, run `npm run update-i18n` in the client repo to get the update